config/routes.rb

```diff
@@ -2830,6 +2830,7 @@
     :restful_redirect => {
       :get => %w(
         index
+        provider_url
```

Comment:
I don't really like the new route thing, any reason why this isn't under `index`?

Reply:
@skateman i kept it separate from `index` method because that method expects different params and is being used to redirect to a page based upon model that's passed in. `provider_url` sets a link that is being used by list of Alerts, not sure if it's a good idea to mix the both methods.